### PR TITLE
Add cmake build configuration

### PR DESCRIPTION
**Pull request by p12tic**
cc @simogasp

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# SOFTWARE.
+
+cmake_minimum_required(VERSION 3.11)
+
+project(Clp LANGUAGES C CXX)
+
+set(PROJECT_VERSION 1.16.11)
+set(PROJECT_VERSION_MAJOR 1)
+set(PROJECT_VERSION_MINOR 16)
+set(PROJECT_VERSION_PATCH 11)
```
Comment (simogasp) [inline CMakeLists.txt]:
```suggestion
project(Clp LANGUAGES C CXX VERSION 1.16.11)
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+set(CMAKE_CXX_STANDARD 14)
+
+include(GNUInstallDirs)
+set(INSTALL_INCLUDE_DIR ${CMAKE_INSTALL_INCLUDEDIR}/Coin)
+set(INSTALL_CMAKE_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
+
+if(MSVC)
```
Comment (simogasp) [inline CMakeLists.txt:182]: same comment as coinutils, only if you want

**Review by simogasp — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (p12tic) [reply CMakeLists.txt:182]: Fixed now.

File: CMakeLists.txt (hunk 1 above)
Comment (p12tic) [reply CMakeLists.txt]: Fixed now.

**Comment by p12tic** [comment]
@simogasp I've addressed your comments.

**Review by simogasp — APPROVED** [review]

**Pull request merged**